index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- google fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <!-- end of google fonts -->
+    <!-- fontawesome -->
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"
+      integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <!-- end of fontawesome -->
+    <!-- style css -->
+    <link rel="stylesheet" href="style.css" />
+    <!-- end of style css -->
+    <title>Emmanuel</title>
+  </head>
+  <body>
+    <header class="header">
+      <div class="logo">
+        <a href="index.html"><h1 class="logo">LOGO</h1></a>
+      </div>
+
+      <div class="menu-bar">
+        <i class="fa-solid fa-bars"></i>
+      </div>
+    </header>
+    <main class="hero">
+      <div class="hero-title">
+        <h1>
+          I'm Emmanuel.<br />
+          Grad to see you!
+        </h1>
+      </div>
+      <div class="hero-text">
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hesitate to contact
+          me.
+        </p>
+      </div>
+      <div class="btn">
+        <button class="hero-btn" type="button">let's connect</button>
+      </div>
+      <div class="hero-social-icons">
+        <a href="#"><i class="fa-brands fa-square-twitter"></i></a>
+        <a href="#"><i class="fa-brands fa-linkedin"></i></a>
+        <a href="#"><i class="fa-brands fa-medium"></i></a>
+      </div>
+    </main>
```

Comment:
- [ ] As pointed out earlier, kindly use the section tag for this main part. Section represents a generic standalone section of the document. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.
```suggestion
    <section class="hero">
      .
      .
      .
    </section>
```

- [ ] Kindly use the css display: flex property to layout the elements in this section this will give us a lot of options to align all our elements, plus, it's responsive. ℹ️